bite-size-notes/week-7-a-kalman-filter.md

```diff
@@ -0,0 +1,129 @@
+# Kalman Filter: A Closed Form Solution to State Estimation
+
+### Scope
+- Linear discrete time system dynamics
+### Objectives
+- The implementation of a Kalman filter
+- The derivation of the Kalman filter
+- A brief description of duality
+
+## Introduction: Help, I Don't Know Where my System is!
+Many modern state space control techniques make use of full state feedback, or the practice of using all state variables to determine the appropriate control input. However, unless the dynamics are known exactly or a perfect observer exists (Spoiler alert: they don't), the full state is rarely accessible. Instead, an estimator can be used to predict the values of the unknown state variables based on the observable states, but this introduces some error between the true state and the estimated state. One option for dealing with this error is to frame it as a minimization control problem. In other words, can a controller be designed that drives the error to zero? Fortunately, the answer is yes and one such implementation is the Kalman filter.
+
+## Preliminaries
+Given the following variables and initial conditions
```

Comment:
Detailed explanations of the variables are very helpful in understanding the formulas